### ISSUE TITLE
create tutorial gifs for Goggles

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.mozilla.org#1232]

**Comment by kristinashu** [comment]
We're leaning towards animated gifs instead of a video (https://redpen.io/p/wjdd96d46ae8c63dc3). I can work on this next heartbeat.

**Comment by hannahkane** [comment]
changed description to match the new plan!

**Comment by Pomax** [comment]
this is probably blocked on the UI being implemented so we can build the gifs for each platform/browser combination that people use. Even firefox on mac vs. firefox on windows has a different look and feel, so making sure we generate the gifs relevant to the user will require having a finalised goggles

**Comment by kristinashu** [comment]
I don't think we're going to do gifs anymore (too distracting and maybe not necessary). But we will need to make different images for each browser and OS. Here's an example on Firefox for Mac https://redpen.io/p/qv1c96b69ffec76998

@Pomax you up for swapping out images and some text (bookmark vs toolbar...) depending on which browser and OS a user is on?

**Comment by kristinashu** [comment]
Closing and opening a new ticket in goggles repo https://github.com/mozilla/goggles.webmaker.org/issues/254
